### PR TITLE
feat(lang): Update language

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Updates the language in the app with following points:
  - Language has been split into their own json. currently only has `en.json`
  - Changed how language is loaded. Now calling  `initLocale` is not needed anymore
  - Updated various fixed text to use translatables instead

### CODE REVIEW

**Review by stavares843 — COMMENTED** [review]
Captura de ecrã 2024-08-01, às 16 33 24

Captura de ecrã 2024-08-01, às 16 33 33

missing these on /settings/preferences

**Review by stavares843 — COMMENTED** [review]
Captura de ecrã 2024-08-01, às 16 34 14

missing this on /settings/gamepad

**Review by stavares843 — COMMENTED** [review]
missing this on /settings/profile

Captura de ecrã 2024-08-01, às 16 36 27

**Comment by luisecm** [comment]
Found these bugs through automation:

Settings Audio - Output and input devices descriptions are backwards
image

Settings Developer - Test Voice description is missing:
image

File: src/routes/settings/developer/+page.svelte [hunk 1]
```diff
@@ -60,18 +58,22 @@
                         }, 500)
                     })
                     .catch(error => log.error(`Error deleting database: ${error}`))
-            }}>Clear State</Button>
+            }}>{$_("settings.developer.clearStateButton")}</Button>
     </SettingSection>
 
-    <SettingSection hook="section-test-voice" name="Test Voice" description="Dev Voice">
-        <Button hook="button-test-voice" appearance={Appearance.Alt} on:click={_ => goto("/developer/debug/voice")}>Voice Dev</Button>
+    <SettingSection hook="section-test-voice" name={$_("settings.developer.testVoice")} description={$_("settings.developer.testVoiceDesciption")}>
```
Comment (stavares843) [inline src/routes/settings/developer/+page.svelte:64]: typo

testVoiceDesciption vs testVoiceDescription

**Review by InfamousVague — APPROVED** [review]

**Comment by InfamousVague** [comment]
This is a lot of work, thanks for this one

**Review by stavares843 — APPROVED** [review]

**Comment by stavares843** [comment]
very neat 🎉

**Pull request merged**